schema/core-im/core-im-source.yaml

```diff
@@ -76,27 +83,44 @@ $defs:
         - $ref: "#/$defs/Method"
         - $ref: "/ga4gh/schema/gks-common/1.x/data-types/json/IRI"
         description: >-
-          A :ref:`Method` that describes all or part of the process through which the information was
-          generated.
+          A specification that describes all or part of the process that led to creation of the 
+          Information Entity (e.g. a specific experimental protocol or data analysis specification 
+          that describe how data were generated, or an evidence interpretation guideline that 
+          describes steps taken to interpret data in making a variant pathogenicity classification).
+        comments: >-
+          Note that this attribute captures specific *instances* of specifications/methods (e.g. the specific
+          electron microscopy method described in https://doi.org/10.1002/cpz1.1045) - as opposed to reporting 
+          a *type* of method applied (e.g. 'Transmission Electron Microscopy').          
       contributions:
         type: array
         ordered: false
         items:
           $ref: "#/$defs/Contribution"
         description: >-
-          A list of :ref:`Contribution` objects that describe the activities performed by agents upon this entity.
-      isReportedIn:
+          Specific actions taken by an Agent toward the creation, modification, validation, or 
+          deprecation of an Information Entity.
+        comments: >-
+          This attribute holds one or more Contribution objects, which provide structured descriptions
+          of a contribution made to the Information Entity by a particular agent.   
+      reportedIn:
```

Comment:
@mbrush this property name change caught me off guard. I'm hoping we won't be making many more changes especially after things get to trial use. I'll accept and make the changes in the pilot implementations.